### ISSUE TITLE
Do not depend on meta-package(s)

### ISSUE THREAD

**Issue by jschleicher** [PilzDE/pilz_robots#61]
prbt_moveit_config depends on ros_control and on ros_controllers, but according to [REP-140](http://www.ros.org/reps/rep-0140.html#metapackage) this is not allowed (see discussion in https://github.com/catkin/catkin_tools/issues/370)

For me the current situation causes error outputs of catkin build:
```
Abandoned <<< prbt_gazebo                                   [ Depends on unknown jobs: ros_control ]                                     
Abandoned <<< prbt_moveit_config                            [ Depends on unknown jobs: ros_control ] 
```
and in some (not reproducible) situation @JonathanGruner experienced an error in the first step of `catkin_make` so that the build failed completely.

**Comment by agutenkunst** [comment]
Can you post the catkin build configuration?

**Comment by jschleicher** [comment]
I can confirm your fix: ff3bd28 doesn't show the "Abondoned" warning any longer.
